### PR TITLE
updates to the TinyMCE quick start

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: quick-start.md [hunk 1]
```diff
@@ -7,55 +7,82 @@ description: Get an instance of TinyMCE up and running in less than 5 minutes.
...
+
+Signing up for a {{site.cloudname}} API key will also provide a trial of the [Premium Plugins]({{site.baseurl}}/enterprise/).
 
 ## Step 3: Saving content with a form POST
+Process the content with a form handler.
+
+When the `<form>` is submitted, the {{site.productname}} {{site.productmajorversion}} editor mimics the behavior of a normal HTML `<textarea>` during the `POST`. The host's form handler can process the submitted content in the same way as content from a regular `<textarea>`.
 
-When the `<form>` is submitted, the {{site.productname}} {{site.productmajorversion}} editor mimics the behavior of a normal HTML `<textarea>` during the `POST`. In the user's form handler, the content submitted can be processed in the same way as the content created from a regular `<textarea>`.
+The HTML elements and inline CSS are also submitted, along with pasted media such as images. Images are submitted as inline content with base64 encoding.
```
Comment (lnewson) [inline quick-start.md]: This isn't quite true for the base64 comment, it's only for images that haven't been uploaded, so that could maybe be confusing?

File: quick-start.md [hunk 2]
```diff
@@ -7,55 +7,82 @@ description: Get an instance of TinyMCE up and running in less than 5 minutes.
...
 
-Since {{site.productname}} enables identifying replaceable elements via a CSS selector, the only requirement is to pass an object that contains a `selector` to `tinymce.init()`.
+**For example**:
 
-In this example, replace `<textarea id='mytextarea'>` with a {{site.productname}} {{site.productmajorversion}} editor instance by passing the selector `'#mytextarea'` to `tinymce.init()`.
+To replace `<textarea id='mytextarea'>` with a {{site.productname}} {{site.productmajorversion}} editor instance, pass the selector `'#mytextarea'` to `tinymce.init()`.
 
 ```html
 <!DOCTYPE html>
 <html>
```
Comment (lnewson) [inline quick-start.md]: We probably should include the `lang="en"` attribute here I guess, since HTML validators normally complain about it? Also should this contain the meta viewport header required for mobile?

File: quick-start.md (hunk 1 above)
Comment (tylerkelly13) [reply quick-start.md]: fair. should I delete the last sentence?

Comment (lnewson) [reply quick-start.md]: It might be easier than trying to explain it in a quick start, but I'm not sure honestly.

File: quick-start.md (hunk 2 above)
Comment (tylerkelly13) [reply quick-start.md]: updated

File: quick-start.md (hunk 1 above)
Comment (tylerkelly13) [reply quick-start.md]: removed

File: general-configuration-guide/use-tinymce-distraction-free.md [hunk 3]
```diff
@@ -6,7 +6,9 @@ description: Theme that renders a lightweight UI for inline editing.
 keywords: theme inlite distraction-free
 ---
 
-The **Distraction-free** theme renders the editor in lightweight inline mode. This mode provides options to quickly insert links, images, and tables into the content. For a sample view of this mode, see the [Distraction-free editor]({{ site.baseurl }}/demo/editor-dfree/) example page.
+The **Distraction-free** theme renders the editor in lightweight inline mode. This mode provides options to quickly insert links, images, and tables into the content.
```
Comment (SimonFc) [inline general-configuration-guide/use-tinymce-distraction-free.md]: Should we avoid calling distraction-free a "theme" and just go with mode?

File: quick-start.md [hunk 4]
```diff
@@ -7,55 +7,82 @@ description: Get an instance of TinyMCE up and running in less than 5 minutes.
...
 <script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
 ```
 
-> Tip: A complete HTML snippet is provided in Step 2.
+## Step 2: Initialize TinyMCE as part of a web form
 
+Initialize {{site.productname}} {{site.productmajorversion}} on any element (or elements) on the web page.
 
-## Step 2: Initialize TinyMCE as part of a web form
+{{site.productname}} enables identifying replaceable elements using a [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors). Specify the element by passing an object that contains a `selector` to `tinymce.init()`, such as:
```
Comment (blackcathikari) [inline quick-start.md]: This and the para above feels a bit awkward. The "identifying replaceable elements" and such seems a bit hard to parse. Could we combine them and simplify it a bit? E.g. 

`Initialize {{site.productname}} {{site.productmajorversion}} on any element (or elements) on the web page by passing an object that contains a `selector` value to `tinymce.init()`. The `selector` value can be any valid [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors).`

File: quick-start.md [hunk 5]
```diff
@@ -7,55 +7,82 @@ description: Get an instance of TinyMCE up and running in less than 5 minutes.
...
 
+Adding this content to a HTML file and opening it in a web browser will load a TinyMCE editor, such as:
+
+{% include codepen.html id="default" %}
+
+To remove the notice:
+
+<img alt="warning icon" src="{{ site.baseurl }}/images/icons/warning.svg" style="vertical-align:middle;"/>**_This domain is not registered with {{site.cloudname}}..._**
+
+Replace `no-api-key` in the source script (`<script src=...`) with a [{{site.cloudname}} API key]({{site.accountsignup}}).
```
Comment (blackcathikari) [inline quick-start.md]: Should this say "...which you can get by signing up..." or something? The next sentence implies it but wondering if we should be more... straightforward?

File: quick-start.md (hunk 1 above)
Comment (blackcathikari) [reply quick-start.md]: I... find this whole sentence quite confusing tbh. Could we try a different angle like "All editor content is submitted including all HTML elements, inline CSS and images."? Although I think it's "submitted" here that's really bugging me since that seems... technically incorrect?

File: general-configuration-guide/use-tinymce-distraction-free.md (hunk 3 above)
Comment (tylerkelly13) [reply general-configuration-guide/use-tinymce-distraction-free.md]: fixed

File: general-configuration-guide/use-tinymce-distraction-free.md [hunk 6]
```diff
@@ -75,12 +77,14 @@ tinymce.init({
...
-* [skin]({{ site.baseurl }}/configure/editor-appearance/#skin)
-* [theme]({{ site.baseurl }}/configure/editor-appearance/#theme)
-
+For information on:
+* Configuring a Quick Toolbar; including `quicklink`, `quickimage`, and `quicktable`; see: [Quick Toolbar plugin]({{site.baseurl}}/plugins/quickbars/).
+* Configuring a contextual toolbar, see: [Context Toolbar]({{site.baseurl}}/ui-components/contexttoolbar/).
+* Configuring a contextual menu, see: [Context Menu]({{site.baseurl}}/ui-components/contextmenu/).
+* Disabling the {{site.productname}} menubar, see: [Menubar]({{site.baseurl}}/configure/editor-appearance/#menubar).
+* The `quickbars_insert_toolbar` setting, see: [quickbars_insert_toolbar]({{site.baseurl}}/configure/editor-appearance/#quickbars_insert_toolbar/).
+* The `quickbars_selection_toolbar` setting, see: [quickbars_selection_toolbar]({{ site.baseurl }}/configure/editor-appearance/#quickbars_selection_toolbar).
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply general-configuration-guide/use-tinymce-distraction-free.md]: yeah, I spotted that too. I think I'll fix this up when I start revising the doc set (it will break links, etc.) I just want to remove the strange and incomplete list at the bottom of this page for now.

File: general-configuration-guide/use-tinymce-distraction-free.md [hunk 7]
```diff
@@ -6,7 +6,9 @@ description: Theme that renders a lightweight UI for inline editing.
 keywords: theme inlite distraction-free
 ---
 
-The **Distraction-free** theme renders the editor in lightweight inline mode. This mode provides options to quickly insert links, images, and tables into the content. For a sample view of this mode, see the [Distraction-free editor]({{ site.baseurl }}/demo/editor-dfree/) example page.
+The **Distraction-free** theme renders the editor in lightweight inline mode. This mode provides options to quickly insert links, images, and tables into the content.
+
+For a sample view of this mode, see the [Distraction-free editor]({{ site.baseurl }}/demo/editor-dfree/) example page.
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply general-configuration-guide/use-tinymce-distraction-free.md]: Agreed

[the remark this answers is not in the document]
Comment (tylerkelly13) [reply general-configuration-guide/use-tinymce-distraction-free.md]: Replaced with: 
To try out a {{site.productname}} editor in distraction-free mode,
Also updated on the inline mode page

File: general-configuration-guide/use-tinymce-distraction-free.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply general-configuration-guide/use-tinymce-distraction-free.md]: scratch that. Including the setting info on the quickbar plugin and deleted these items

File: general-configuration-guide/use-tinymce-inline.md [hunk 8]
```diff
@@ -11,7 +11,9 @@ keywords: form inline edit stylesheet
 
 The inline editing mode is useful when creating user experiences where the editing view of the page can be merged with the reading view of the page. This creates a seamless editing experience and true WYSIWYG behavior.
 
-When in inline editing mode the editor does not replace the selected element with its iframe, but instead edits the element's content in place instead. For a sample view of this mode, visit the [Inline Editor]({{ site.baseurl }}/demo/inline) example page.
+When in inline editing mode the editor does not replace the selected element with its iframe, but instead edits the element's content in place instead.
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply general-configuration-guide/use-tinymce-inline.md]: subbed

File: quick-start.md (hunk 4 above)
Comment (tylerkelly13) [reply quick-start.md]: fixed

File: quick-start.md (hunk 5 above)
Comment (tylerkelly13) [reply quick-start.md]: fixed?
